docs/hardware/devices/heltec/index.mdx

```diff
@@ -61,7 +61,7 @@ This device may have issues charging a connected battery if utilizing a USB-C to
 :::
 
 - **MCU:**
-  - ESP32-S3FN8 (WiFi & Bluetooth)
+  - ESP32-S3FN8 (WiFi & Bluetooth, but not both at the same time. Only enable one at a time.)
```

Comment:
The MCU support both, but not at the same time. Without saying it here (where people will be expecting all the necessary information about the V3), people will not understand that they need to disable wifi if they want bluetooth to work, or vice-versa.

If all esp32 devices are like this, then let's say it on every page.

Reply:
Rather than merely describe a problem, you should give its solution. "but not both at the same time" describes the problem. "Only enable one at a time" is the solution. I think the text is fine as I've written it.